css/blocks/contents.css

```diff
@@ -0,0 +1,54 @@
+.contents {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 30px;
+  margin: 0px;
+  margin-bottom: 94px;
+  padding: 0px;
+  list-style: none;
+}
+
+.contents__label {
+
+  width: 370px;
+  height: 404px;
+  outline: 1px solid #EEE;
+  background: #FFF;
+}
+
+.contents__image {
+  width: 370px;
+  height: 294px;
+  flex-shrink: 0;
+}
+
+.contents__title {
+  margin-top: 20px;
+  margin-bottom: 0px;
+  margin-left: 24px;
+  padding: 0px;
+  color: #212121;
+  font-family: Roboto;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 36px;
+  letter-spacing: 1.08px;
+}
+
+.contents__option {
+
+  margin-top: 0px;
+  margin-bottom: 30px;
+  margin-left: 24px;
+  padding: 0px;
+  color: rgba(117, 117, 117, 1);
+  font-family: Roboto;
```

Comment:
do not repeat yourself - font-family setting in the `html` tag is sufficient, no need to repeat it, except for tag `a` that doesn't inherit font-family from a parent element